Firebase.podspec

```diff
@@ -62,21 +62,21 @@ Simplify your app development, grow your user base, and monetize more effectivel
   end
 
   s.subspec 'Analytics' do |ss|
-    ss.ios.deployment_target = '9.0'
+    ss.ios.deployment_target = '10.0'
```

Comment:
Change line 36 also, but not 46 (Crashlytics)

Reply:
Done in [c34fdfe](https://github.com/firebase/firebase-ios-sdk/pull/9601/commits/c34fdfe77643f234fdfa57100a1c1c76ba0fccb6)